wagtail/admin/tests/api/test_pages.py

```diff
@@ -101,6 +103,20 @@ def test_private_pages_appear_in_list(self):
         content = json.loads(response.content.decode('UTF-8'))
         self.assertEqual(content['meta']['total_count'], new_total_count)
 
+    def test_get_in_non_content_language(self):
+        # set logged-in user's admin UI language to Swedish
+        user = get_user_model().objects.get(email='test@email.com')
+        UserProfile.objects.update_or_create(user=user, defaults={'preferred_language': 'se'})
+
+        response = self.get_response()
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response['Content-type'], 'application/json')
+
+        # Will crash if the JSON is invalid
+        content = json.loads(response.content.decode('UTF-8'))
```

Comment:
FWIW, I believe a crash will result in an error rather than a failure which isn't quite the same thing (or is it only pytest making that difference?) but at the end of the day, the test will fail and that's what we want.